### DISCUSSION TITLE
Where to open / see known issues ?

### DISCUSSION THREAD

**Discussion by maxandersen** [iongion/container-desktop#7] [General]
Hi,

I see issues are turned off so wondered where one see existing known issues and where to report new ones found ?

**Comment by gbraad** [comment]
I wanted to start a conversation as a 'enhancement' request. Discussions aren't my preferred solution for that as they aren't linked to the PR if submitted.

I have emailed the author about 2 weeks ago. Most likely he is on PTO, but the project also seems to be ... ? A shame as it prevents the project to grow.

**Comment by iongion** [comment]
Hi guys, I am going to open the issues once podman supports native volumes mapping for Windows and MacOS, it is the only reason I don't open it as there will be so many complaining about it. Development is paused until podman `4.0` reaches stability. I do consider these as critical for cross operating system availability.

**Comment by afbjorklund** [comment]
> once podman supports native volumes mapping for Windows and MacOS

As far as I know, WSL2 has mappings from the Windows file system in the `/mnt` directory: https://docs.microsoft.com/en-us/windows/wsl/faq

```
/mnt/c/Users/
```

It is possible to set something similar up with the new `-v` flag to `podman machine init`, but at the moment it requires _patching_ Mac QEMU.

```
-v /Users:/mnt/Users
```

It is always possible to use `ssh` to access the file system on the VM:

* https://github.com/containers/podman/issues/8016#issuecomment-998777418

Including using `sshocker` (reverse sshfs), the way it is being done in Lima ?

But right now (`4.0.0-dev`), there are **no** host mounts in the default machine VM.

**Reply by iongion** [reply]
As far as I know, Jason Greene is working on something awesome for WSL, the ability to have native mounts will be there.
He is working on a special WSL distro, that should fix all these and allow native mounts.

Meanwhile, I stopped the work when trying to find a way to communicate using unix socket for both WSL and Lima, but I should pick-it up easily once the full time job pressure of beginning of the year gets smaller.

**Reply by gbraad** [reply]
The mappings in `/mnt` are almost useless. https://github.com/microsoft/WSL/issues/4064
Often times they are out of sync and cause more harm.
SSHFS is way more reliable, and works across the board for the different implementations we have/use.

**Reply by afbjorklund** [reply]
That issue seemed to about file events, which probably don't sync for remote filesystems ? There are some similar reports about stability of sshfs in lima, but it seems related to people trying to keep their entire workspaces on the remote mount and not on the local filesystem.

Even though support for "virtfs" merged now, I still don't think it's a good idea. Neither from a security, nor a performance, point of view. If Docker hasn't got a working solution yet, then it seems unlikely that Podman will. Maybe with virtio-fs, but I am not holding my breath...
